GoMeet/src/main/webapp/test/spec/spec-create-meeting-step1.js

```diff
@@ -0,0 +1,76 @@
+describe ("Add guest function", function() {
+
+  beforeAll(function() {
+    addGuest(document);
+  });
+
+  afterAll(function() {
+    removeGuest('guest-1', document); 
+  });
+
+  it ("Should add one li element to the html page", function() {
+    let listSize = document.getElementById('guest-list').childNodes.length; 
+    let elementType = document.getElementById('guest-1').tagName;
+    expect(listSize).toBe(1); 
+    expect(elementType).toBe("LI"); 
+  });
+
+  it ("Should add a li element with 3 child nodes", function() {
+    let listElement = document.getElementById('guest-1'); 
+    let numChildren = listElement.childNodes.length; 
+    expect(numChildren).toBe(3); 
+  });
+
+  it ("The first child node should be a label element", function() {
+    let children = document.getElementById('guest-1').childNodes; 
+    let elementType = children[0].tagName;
+    expect(elementType).toBe("LABEL"); 
+  });
+
+  it ("The second child node should be an input element", function() {
+    let children = document.getElementById('guest-1').childNodes; 
+    let elementType = children[1].tagName; 
+    expect(elementType).toBe("INPUT");
+  });
+
+  it ("The third child node should be a button element", function() {
+    let children = document.getElementById('guest-1').childNodes; 
+    let elementType = children[2].tagName; 
+    expect(elementType).toBe("BUTTON");
+  });
```

Comment:
Are these all seperate tests? If so, I think they could be combined as one, and you could add an additional test to add multiple guests.

Reply:
Combined and added the extra test for adding more than one guest :)